### DISCUSSION TITLE
no entry since 12.11.2023

### DISCUSSION THREAD

**Discussion by Harleyluhja** [teslamate-org/teslamate#3822] [General]
Hi, I ask you for help. After a long time, I checked Teslamate again and found that no data has been arriving in the database since November 12, 2023. Now I updated the database with docker-compose down docker-compose pull and docker-compose up -d. Unfortunately Teslamate still doesn't work. Do you have an idea.

**Comment by cwanja** [comment]
What do the logs say?
